ASUWebPlatforms/webspark-ci
# WS2-1598 Add accordion default open functionality/setting

### Description

I added a field to each accordion_panel paragraph and hid them on all but the first instance. I spent a lot of time trying to figure out how to do this via Twig, but found it too unwieldy, so resorted to JS.

QA Test Steps:

Visit /accordion while logged in as an administrator or editor

Click on “Layout” and then click on the pencil icon and “Configure” for one of the accordion blocks

In the dialog that opens, note that a new checkbox field is available on the first accordion item that says “Initially Expanded.”

Also note that this field does not appear on any of the other accordion items. 

Click the checkbox and save

The first accordion item should now be expanded by default. If so, this passes.
### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1598)

### Checklist


### Verified in browsers 


### Screenshots



Note: Sections that are not applicable for this issue can be removed.
